spec/models/intake/gyr_intake_spec.rb

```diff
@@ -292,23 +292,65 @@
   end
 
   describe "#duplicates" do
-    context "when hashed_primary_ssn is nil" do
-      let(:intake) { create :intake, primary_ssn: nil }
+    context "when an itin applicant" do
+      let(:dupe_double) { double }
+      let(:triage) { create(:triage, id_type: "need_itin_help" )}
+      before do
+        allow(DeduplificationService).to receive(:duplicates).and_return dupe_double
+        allow(dupe_double).to receive(:or)
+      end
+      context "when only email is present" do
+        let(:intake) { create :intake, primary_birth_date: Date.tomorrow, email_address: "mango@example.com", sms_phone_number: nil, triage: triage }
+        it "responds with duplicates from birth date and email" do
+          intake.duplicates
+          expect(DeduplificationService).to have_received(:duplicates).exactly(1).times.with(intake, :email_address, :primary_birth_date, from_scope: described_class.accessible_intakes)
+        end
+      end
+
+      context "when only sms is present" do
+        let(:intake) { create :intake, primary_birth_date: Date.tomorrow, email_address: nil, sms_phone_number: "+18324658840", triage: triage }
+        it "responds with duplicates from sms" do
+          intake.duplicates
+          expect(DeduplificationService).to have_received(:duplicates).exactly(1).times.with(intake, :sms_phone_number, :primary_birth_date, from_scope: described_class.accessible_intakes)
+        end
+      end
+
+      context "when both email and sms are present" do
```

Comment:
Should you check `phone_number` in addition to `email` and `sms_phone_number`? I think so.

Reply:
no, i dont think so.... phone number is where people take the intake call. Many people use shared phones for this.

Reply:
This is a very good point. Thanks for the reminder. :)